hackernews/program.py

```diff
@@ -0,0 +1,45 @@
+import os
+import requests
+import time
+import urllib.parse
+
+from autokitteh.slack import slack_client
+
+
+slack = slack_client("slack_connection")
+SLACK_CHANNEL = os.getenv("SLACK_CHANNEL")
+
+
+def on_command(event):
+    topic = event.data.text  # Extract the topic from the Slack command
```

Comment:
if user enters multiple words, will that affect the workflow in any significant way? will it work the same?

Reply:
it will work the same. it will take the query encode it (the spaces) and add it to the search query